rust_src/remacs-lib/docfile.rs

```diff
@@ -27,12 +27,12 @@ const FUNCTION: c_int = 5;
 type AddGlobalFn = fn(c_int, *const c_char, c_int, *const c_char) -> *const ();
 
 #[no_mangle]
-pub extern "C" fn scan_rust_file(
+pub unsafe extern "C" fn scan_rust_file(
```

Comment:
I'm curious about this -- isn't it better to limit `unsafe` to the specific line?

Reply:
Clippy says no.....

Reply:
Which makes sense. If that `unsafe` line hits a snag the code is going to crash.